examples/websets/import_example.ts

```diff
@@ -0,0 +1,130 @@
+/**
+ * Import Example
+ *
+ * This example shows how to create an import and upload CSV data to Websets.
+ */
+
+import * as dotenv from "dotenv";
+import Exa, { CreateImportParametersFormat } from "../../src";
+
+// Load environment variables from .env file
+dotenv.config();
+
+async function main() {
+  // Initialize the Exa client
+  const apiKey = process.env.EXA_API_KEY;
+  if (!apiKey) {
+    console.log("Please set EXA_API_KEY environment variable");
+    console.log("Example: export EXA_API_KEY='your-api-key-here'");
+    process.exit(1);
+  }
+
+  const exa = new Exa(apiKey);
+
+  // Sample CSV data for companies
+  const csvData = `company_name,website_url,location
+Apple Inc,https://www.apple.com,Cupertino California
+Google LLC,https://www.google.com,Mountain View California
+OpenAI,https://openai.com,San Francisco California
+Microsoft Corporation,https://www.microsoft.com,Redmond Washington
+Meta Platforms,https://www.meta.com,Menlo Park California`;
+
+  try {
+    console.log("Creating import...");
+
+    // Create an import for company data
+    const importResponse = await exa.websets.imports.create({
+      title: "Sample Company Data",
+      format: CreateImportParametersFormat.csv,
+      entity: { type: "company" },
+      size: Math.ceil(Buffer.byteLength(csvData, "utf8") / (1024 * 1024)) || 1, // Size in MB, minimum 1
+      count: 5,
+    });
+
+    console.log(`✓ Created import: ${importResponse.id}`);
+    console.log(`  Upload URL valid until: ${importResponse.uploadValidUntil}`);
+
+    // Upload the CSV data with proper S3 headers
+    console.log("\nUploading CSV data...");
+    const csvBuffer = Buffer.from(csvData, "utf8");
+
+    const uploadResponse = await fetch(importResponse.uploadUrl, {
+      method: "PUT",
+      headers: {
+        "Content-Length": csvBuffer.length.toString(),
+      },
+      body: csvBuffer,
+    });
```

Comment:
(I'm starting with the example so haven't read the actual code yet so this may be wrong), but is there a reason to have this be two calls? I think in API it makes sense, but for client I could see it being nice if there was a single method that just took the definition and a path to the csv (i.e. also could be nice if you don't have to think about  the length of the file yourself, and will get just get an error if it's too big / small)